### PR TITLE
recipes

### PR DESCRIPTION
**Pull request by RaviTella**
Added both read and write recipes

### CODE REVIEW

File: cosmos/sql/recipes/ycsb/build-your-own/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "Specifies the location for all resources."
+            }
+        },
+        "deploymentName": {
+            "type": "string",
+            "defaultValue": "[deployment().name]"
```
Comment (simplynaveen20) [inline cosmos/sql/recipes/ycsb/build-your-own/azuredeploy.json:21]: I assume we will work on hiding this in separate PR

File: cosmos/sql/recipes/ycsb/read/100-thousand-rps-read/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+        "vnetSubnetAddressPrefix": {
+            "type": "string",
+            "defaultValue": "10.2.0.0/24",
+            "metadata": {
+                "description": "Specifies a Virtual Network Subnet Address Prefix "
+            }
+        },
+        "threads": {
+            "type": "int",
+            "defaultValue": 115,
```
Comment (simplynaveen20) [inline cosmos/sql/recipes/ycsb/read/100-thousand-rps-read/azuredeploy.json:94]: this should be 230 , 10 times that of 10k ?

File: cosmos/sql/recipes/ycsb/write/500-thousand-rps-write/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+        "vmSize": {
+            "type": "string",
+            "defaultValue": "Standard_D48s_v4",
+            "metadata": {
+                "description": "Specifies the number of Virtual Machines/clients"
+            }
+        },
+        "vmCount": {
+            "type": "int",
+            "defaultValue": 2,
```
Comment (simplynaveen20) [inline cosmos/sql/recipes/ycsb/write/500-thousand-rps-write/azuredeploy.json:46]: This should be 10 ?

Comment (RaviTella) [reply cosmos/sql/recipes/ycsb/write/500-thousand-rps-write/azuredeploy.json]: Good catch. Fixed it.

File: cosmos/sql/recipes/ycsb/build-your-own/azuredeploy.json (hunk 1 above)
Comment (RaviTella) [reply cosmos/sql/recipes/ycsb/build-your-own/azuredeploy.json]: Good Catch. I missed this in build-your-own recipe.  Fixed it.

File: cosmos/sql/recipes/ycsb/read/100-thousand-rps-read/azuredeploy.json (hunk 2 above)
Comment (RaviTella) [reply cosmos/sql/recipes/ycsb/read/100-thousand-rps-read/azuredeploy.json:94]: This recipe uses 2 VMs each with 115 threads. Calculation is based 2.3 ms per read.

**Review by simplynaveen20 — APPROVED** [review]
LGTM

**Pull request merged**